lib/internal/modules/run_main.js

```diff
@@ -95,14 +95,17 @@ function shouldUseESMLoader(mainPath) {
 
 /**
  * Run the main entry point through the ESM Loader.
- * @param {string} mainPath - Absolute path for the main entry point
+ * @param {URL} mainPath - Absolute path for the main entry point
```

Comment:
Was the previous type `string` incorrect, or was it unintentional to change the type?

Reply:
Thanks, fixed.